dids/diddht/internal/bep44/bep44_test.go

```diff
@@ -0,0 +1,63 @@
+package bep44
```

Comment:
what's the convention on the package of tests being including `_test`. i imagine it has something to do with whether you're testing private functions. generally prefer using `_test` when possible because it gives me a chance to try out the api surface from a consumer's perspective

Reply:
my habit is to put tests in the same package; reason being, among sometimes wanting to test individual private parts of the package, it also gives access to some of the internals of the structs; so if I want to assert a certain part of the logic is doing the steps that I expect it to do, I can. tests in the same package have more flexibility in that sense, while not excluding the ability to test the API. same package is a more unit-test-y approach, which is what I'm most used to, while the _test is more "integration" kind. I can't say they necessarily exclude each other